core/src/banking_stage.rs

```diff
@@ -1267,7 +1270,7 @@ impl BankingStage {
                     bank.commit_transactions(
                         sanitized_txs,
                         &mut loaded_transactions,
-                        execution_results,
+                        execution_results.clone(),
```

Comment:
This `clone()` should be unnecessary, it looks like `execution_results` gets passed back out as `tx_results.execution_results`?

Reply:
Yea, but later in same closure, `transaction_status_sender` consumes the `tx_results.execution_results`. I didn't want to dig down that path early, but let me see what can be done. Will make this change in next commit.

Reply:
Agreed with @ckamm, the qos tracker just needs a list of booleans that say whether the transaction was committed to the block or not, cloning here is overkill.

Reply:
A list of `executed_indexes` could work too

Reply:
[commit](d34a42ac85d3468edfbc4143a1066a487d80087d) extract `was_executed()` from execution_result into vec<bool>.